vector/src/test/java/im/vector/app/features/settings/devices/v2/DevicesViewModelTest.kt

```diff
@@ -72,6 +73,7 @@ class DevicesViewModelTest {
     private val fakeInterceptSignoutFlowResponseUseCase = mockk<InterceptSignoutFlowResponseUseCase>()
     private val fakePendingAuthHandler = FakePendingAuthHandler()
     private val fakeRefreshDevicesUseCase = mockk<RefreshDevicesUseCase>(relaxUnitFun = true)
+    private val fakeSharedPreferences = FakeSharedPreferences()
```

Comment:
New tests should be added:
- to cover the new toggle visibility action
- to cover the update of viewState at the initialization